### DISCUSSION TITLE
How to revert accent override to pre-2.2.0 behavior

### DISCUSSION THREAD

**Discussion by 7kt4** [Maplespe/DWMBlurGlass#310] [General]
I would like to compile the source code with the accent override method that was used before 2.2.0 before it had the double-painting and corners issue. (Not talking about the shader, I mean [this](https://github.com/Maplespe/DWMBlurGlass/issues/289)) But I have been searching through the source code and can't find the file responsible for this. Can someone guide me?

And, I would suggest to revert the behavior in the main branch, since the new accent overriding method brings only bugs but not any benefits. (Not overriding the accent blur of taskbar thumbnails/jump list, only of the taskbar and start menu)

**Comment by Maplespe** [comment]
In fact, we never changed the code, and 2.2.0's override of Accent colors functions the same as 2.1.1.
You can check our commit log and no changes have been made to the code related to overriding Accent.

**Reply by Maplespe** [reply]
Also, only AeroBackdrop has this problem, the others (Blur, Acrylic, Mica) do not.
So it might have something to do with the new AeroBackdrop's layer, I don't know because I didn't write that part.

**Comment by TheInsane101** [comment]
I disagree. I look at the taskbar and start menu far more often than the thumbnails. Plus, the thumbnail borders are much smaller than the entire taskbar and start menu. 
On top of that, not everyone universally uses thumbnails as often, especially if they have taskbar programs ungrouped, but everyone universally sees the taskbar and start menu much more often, therefore an inaccurately shadered TB and SM will be far more noticeable and bothersome than inaccurate thumbnails. 

Finally, the thumbnails and jumplists will never be as accurate as 7's, since we haven't found how to restore the exact way in which the thumbnails work in 7 (7's thumbnails and jump lists have the shine texture and all thumbnails share the same size of the coloured background/border, with only the window preview being its own size. In 8 and newer, the size of the window changes the size of the background and border). Meanwhile, with the current shader, the taskbar and start menu are exactly 1-1 with 7. Reverting will leave _both_ UI components incompletely accurate.

I'm happier with what we have now, so much so that I'm happy to put up with having to restart explorer to fix the layer issue everytime I reboot Windows or DWM crashes. I don't know how the devs will be able to make SIB use the old shader and the rest of the UI use the new shader.

**Comment by kfh83** [comment]
I honestly just hope someone comes up with a competent SIB replacement because quite frankly it sucks (on top of being completely closed source)

**Reply by TheInsane101** [reply]
Crazy to think how it and Classic/Open-Shell have been the consistent to-go-tos for over 11 years, and that they are 11 years old already.

**Reply by kfh83** [reply]
Classic/Open-shell have even more glaring inaccuracies and i seriously don't get anyone who use it (atleast for 7 setups), SIB is bad but atleast it manages to be in the ballpark, not to mention the former literally can't theme the taskbar unless you pair it with SIB
